_posts/2018-10-01-strimzi-0.7.0-release.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title:  "Strimzi 0.7.0 released!"
+date: 2018-10-01
+author: paolo_patierno
+---
+
+We are delighted to announce the new Strimzi 0.7.0 release with many awesome new features!
+
+<!--more-->
+
+# Exposing Kafka cluster to the outside
+
+The Strimzi community has been asking for it for long time and finally it's here!
+The Kafka cluster is now accessible from the outside of the Kubernets/OpenShift cluster where it's deployed.
+Your application doesn't need to be containarized and running in the same Kubernetes/OpenShift cluster for connecting to the Kafka cluster; your application can be any client running outside of it.
+
+There are three different ways for doing that using:
+
+* OpenShift Routes (so OpenShift only)
+* LoadBalancers
+* NodePorts
+
+In order to do that, the new `external` listener was added to the `Kafka` resource.
+It can be of three different types : `route`, `loadbalancer` and `nodeport`.
+
+Here's an example snippet about exposing the Kafka cluster using OpenShift routes:
```

Comment:
Here's an example snippet that exposes the Kafka cluster using OpenShift routes: